pkg/utils/file.go

```diff
@@ -38,7 +38,7 @@ func IsExecutable(filename string) bool {
 		return false
 	}
 	info, _ := os.Stat(filename)
-	return info.Mode()&0x0100 != 0
+	return info.Mode()&0o0100 != 0
```

Comment:
What is this &0o0100?

Reply:
0o0100 (better 0o100, will fix), is a bit mask that, when applying bitwise AND (&) results
in a != 0 state whenever the file can be executed by the owner.
A perm can be seen as a 9-bit sequence mapping to rwxrwxrwx read-write-execute
for owner,group,everybody.
octal 0o100 means binary 001000000.
Probably better to set a constant and also add a comment

Reply:
It previously was 0x0100 == binary 100000000 that was a bug, that was all combinations where
the owner has read perms

Reply:
Gotcha, would you mind adding a comment that explains this?

Reply:
added